src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2096,25 +2134,47 @@ private MethodSymbol InferMethod(BoundCall node, MethodSymbol method, ImmutableA
             // PROTOTYPE(NullableReferenceTypes): Support field initializers in local functions.
         }
 
-        private static BoundExpression RemoveImplicitConversions(BoundExpression expr)
+        /// <summary>
+        /// Returns the expression without the top-most conversion plus the conversion.
+        /// If the expression is not a conversion, returns the original expression plus
+        /// the Identity conversion. If `includeExplicitConversions` is true, implicit and
```

Comment:
>includeExplicitConversions [](start = 41, length = 26)

I'm not sure how to include that in the comment, but things became clearer once I realized the only `VisitConversion` uses `includeExplicitConversions = true`. All other scenarios use `false`.